src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -199,6 +200,16 @@ public class KNNSettings {
      */
     public static final Setting<Boolean> IS_KNN_INDEX_SETTING = Setting.boolSetting(KNN_INDEX, false, IndexScope);
 
+    /**
+     * If this setting is set to True for Faiss Index with SQFP16 Encoder,
+     * then any value in the vector which is outside of FP16 range will be clipped to FP16 range.
+     */
+    public static final Setting<Boolean> IS_KNN_INDEX_FAISS_CLIP_FP16_RANGE_SETTING = Setting.boolSetting(
```

Comment:
I thought that we are having this in index mapping parameter?

Reply:
no, I think based on our discussion it is index setting

Reply:
The value have a meaning only when user use sqfp16. Index setting should have more general setting on index itself.